README.md

```diff
@@ -1436,6 +1437,7 @@ See also [Are we web yet?](http://www.arewewebyet.org) and [Rust web framework c
   * [seed](https://seed-rs.org/) — A Rust framework for creating web apps
   * [stdweb](https://crates.io/crates/stdweb) — A standard library for the client-side Web [![Build Status](https://api.travis-ci.org/koute/stdweb.svg?branch=master)](https://travis-ci.org/koute/stdweb)
   * [yew](https://crates.io/crates/yew) — Rust framework for making client web apps
+  * [sauron](https://github.com/ivanceras/sauron) - Client side web framework which closely adheres to The Elm Architecture. [![Build Status](https://travis-ci.org/ivanceras/sauron.svg?branch=master)](https://travis-ci.org/ivanceras/sauron)
```

Comment:
The image link should be https://api.travis-ci.org/ivanceras/sauron.svg?branch=master as the current one 302's to that

Reply:
I can't reproduce the 302 error. Please check again.

Reply:
So, it's actually a 301. `curl -v https://travis-ci.org/ivanceras/sauron.svg?branch=master` gets a `HTTP/1.1 301 Moved Permanently` with ` Location: https://api.travis-ci.org/ivanceras/sauron.svg?branch=master`

Reply:
Ah thanks, I didn't realize that by just opening the links using the browser.